ambry-network/src/main/java/com/github/ambry/network/http2/Http2BlockingChannel.java

```diff
@@ -44,13 +44,17 @@
  */
 public class Http2BlockingChannel implements ConnectedChannel {
   private static final Logger logger = LoggerFactory.getLogger(Http2BlockingChannel.class);
-  private final Http2MultiplexedChannelPool http2MultiplexedChannelPool;
+  private final ChannelPool channelPool;
   private final Http2ClientConfig http2ClientConfig;
-  final static AttributeKey<Promise<ByteBuf>> RESPONSE_PROMISE = AttributeKey.newInstance("ResponsePromise");
-
-  public Http2BlockingChannel(Http2MultiplexedChannelPool http2MultiplexedChannelPool) {
-    this.http2MultiplexedChannelPool = http2MultiplexedChannelPool;
-    this.http2ClientConfig = http2MultiplexedChannelPool.getHttp2ClientConfig();
+  private final InetSocketAddress inetSocketAddress;
+  final static AttributeKey<CompletableFuture<ByteBuf>> RESPONSE_PROMISE = AttributeKey.newInstance("ResponsePromise");
+  final static AttributeKey<ChannelPool> CHANNEL_POOL_ATTRIBUTE_KEY = AttributeKey.newInstance("channelPool");
+
+  public Http2BlockingChannel(ChannelPool channelPool, InetSocketAddress inetSocketAddress,
```

Comment:
Test code now also use this constructor. Do you mind remove the other one?

Reply:
It's used in the integration test.